compiler-rt/lib/scudo/standalone/common.h

```diff
@@ -112,6 +112,21 @@ template <typename T> inline void shuffle(T *A, u32 N, u32 *RandState) {
   *RandState = State;
 }
 
+inline void computePercentage(uptr Numerator, uptr Denominator, uptr *Integral,
+                              uptr *Fractional) {
+  constexpr uptr Digits = 100;
+  if (Denominator == 0) {
```

Comment:
I'm wondering if we want to assert a non-zero denominator. It seems to give the impression that 0/1 and 0/0 are the same here

Reply:
Another case I saw in real life, and I was wondering if I should say 100% when the denominator is 0. Since 0% feels like the wrong value, 100% also seems kind of wrong, but more correct than 0%. My other thought was doing NA, but that seems like it would be very difficult to implement without changing a lot of things.

I would think setting this to 100% seems the least bad and the most correct, but what do you think?

Reply:
You're right, I think 100% is better! (Given that NA is more complicated)

Reply:
Set to 100, and modified the two locations that use it to reserve enough space for that 100%.